sdk/src/Core/Amazon.Runtime/Internal/Transform/UnmarshallerContext.cs

```diff
@@ -361,6 +363,39 @@ public Stream Stream
             }
         }
 
+        /// <summary>
+        /// For map shapes the KeyXmlName represents the "locationName" on the map shape's key member. This can be changed
+        /// via the @xmlName trait. This ensures that we don't check for a hardcoded "key" value, if the key's name was changed.
+        /// </summary>
+        public string KeyXmlName
```

Comment:
nit: these could be `public string KeyXmlName { get; set; }` without needing member variables.

Reply:
used auto properties